app.js

```diff
@@ -19,11 +20,15 @@ menuBtn.addEventListener('click', () => {
         menuView.classList.remove('slide-out')
         menuView.classList.add('slide-in');
         dim.style.display = 'block';
+        body.classList.remove('no-scroll')
```

Comment:
You added the class list here and removed it in the exact next line.
I'm curious how it works, can you share a short screen record of your solution?